4_Largest_Palindrome_Product.py

```diff
@@ -0,0 +1,27 @@
+
+num1 = 999
+max_palindrome = 0
+while num1 > 0:
```

Comment:
Decrementing ranges might be helpful here.

```python
for x in range(num, -1, -1):
  pass
```

This takes from you x to 0, inclusive